src/main/java/spiderman/Spiderman.java

```diff
@@ -0,0 +1,238 @@
+package spiderman;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+import java.util.ArrayList;
```

Comment:
I think it may be clearer for import statements to be ordered consistently, for example ordering imports from the same package lexicographically (ArrayList should be before Scanner). Supposedly, IDEs can be configured somehow to help you do this, or if you have caught up to week 4, see A-CheckStyle.